src/linux/scripts/bunkerweb-scheduler.sh

```
@@ -135,6 +135,32 @@ function start() {
+    else
+        log "SYSTEMCTL" "⚠️" "Failed to parse DATABASE_URI (error), using defaults"
+        type="(unknown)"
+        database="(none)"
+        host="(local)"
+        user="(none)"
+        ssl_enabled="no"
+        pem_path="(none)"
+    fi
+    db_ssl_enabled="$ssl_enabled"
+    db_pem_path="$pem_path"
+    db_port="$port"
```

Shellcheck (0.11.0)

[warning] port is referenced but not assigned.

(SC2154)